src/main/java/com/hillel/dzhurynskyi/homework/hm4/ex2/interfaces/Participant.java

```diff
@@ -0,0 +1,7 @@
+package com.hillel.dzhurynskyi.homework.hm4.ex2.interfaces;
+
+public interface Participant {
+    boolean run(int size);
```

Comment:
Little confusing names. Run return true/false. Maybe better to use something like 'canRun(int distanceLimit)'